core/src/main/scala/medeia/decoder/BsonDecoder.scala

```diff
@@ -183,7 +195,7 @@ trait DefaultBsonDecoderInstances extends BsonIterableDecoder {
 }
 
 trait BsonIterableDecoder extends LowestPrioDecoderAutoDerivation {
-  def iterableDecoder[A: BsonDecoder, C[_] <: Iterable[A], X](implicit factory: Factory[A, C[A]]): BsonDecoder[C[A]] =
```

Comment:
I do not remember the purpose of this type parameter, so I removed it. Did I forget something? @markus1189

Reply:
I don't recall :thinking: If it works it works :shrug: